charts/cinder-csi-plugin/templates/controllerplugin-statefulset.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- include "cinder-csi.controllerplugin.labels" . | nindent 4 }}
 spec:
+  serviceName: {{ include "cinder-csi.name" . }}-controller-service
```

Comment:
maybe add a dummy service too as it is done in the manila chart?

stateful sets need  a service https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#statefulset-v1-apps
```
serviceName is the name of the service that governs this StatefulSet. This service must exist before the StatefulSet, and is responsible for the network identity of the set. Pods get DNS/hostnames that follow the pattern: pod-specific-string.serviceName.default.svc.cluster.local where "pod-specific-string" is managed by the StatefulSet controller.
```

Reply:
I broke it out here: https://github.com/kubernetes/cloud-provider-openstack/pull/1106

hoping for a fast merge :)

Reply:
this is done in #1105